### PR TITLE
Add stance effect for Stretching Reach

### PR DESCRIPTION
**Pull request by simonward**

### CODE REVIEW

File: packs/feat-effects/stance-stretching-reach.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                "predicate": [
+                    "item:equipped",
+                    "item:melee",
+                    "item:usage:hands:2",
+                    {
+                        "not": "item:trait:reach"
+                    }
+                ],
+                "property": "traits",
+                "value": "reach-10"
```
Comment (TikaelSol) [inline packs/feat-effects/stance-stretching-reach.json:37]: Does this even work on PCs?

Comment (simonward) [reply packs/feat-effects/stance-stretching-reach.json:37]: It seems to when added via `ItemAlteration`. Not sure it's considered valid though? Should I switch to just `"reach"`?

Comment (simonward) [reply packs/feat-effects/stance-stretching-reach.json:37]: Screen Shot 2024-09-12 at 9 10 02 AM

Comment (TikaelSol) [reply packs/feat-effects/stance-stretching-reach.json:37]: Yeah, use reach

**Pull request merged**